src/Microsoft.DotNet.Build.Tasks/Targets/PerfTesting.targets

```diff
@@ -40,7 +39,9 @@
 
   <!-- Executable properties -->
   <PropertyGroup>
-    <XunitRunnerArgs>$(TargetFileName) -runnerhost $(TestHost) -runner $(XunitExecutable) -runid $(PerfRunId) -verbose</XunitRunnerArgs>
+    <PerfRunId Condition="'$(PerfRunId)' == ''">$(TargetFileName)-WindowsCore</PerfRunId>
```

Comment:
Minor note: should we just use $(OS), or do we think running on Linux will probably be different enough that this won't be re-used there?